phoenicis-engines/src/main/java/org/phoenicis/engines/EngineSettingsManager.java

```diff
@@ -53,7 +53,7 @@ public void getSetting(String engineId, String settingId, Consumer<EngineSetting
         final InteractiveScriptSession interactiveScriptSession = scriptInterpreter.createInteractiveSession();
 
         interactiveScriptSession.eval(
-                "include([\"engines\", \"" + engineId + "\", \"settings\", \"" + settingId + "\"]); new Setting();",
+                "include(\"engines." + engineId + ".settings." + settingId + "\"); new Setting();",
```

Comment:
I’m not sure that it is secure. Have you checked that we cannot inject some code?

Reply:
That is a good point. The question is whether we can make the code secure at all?
I mean the previous implementation had the same problem in that you could inject code via the `engineId` and `settingId` parameters, right?

Reply:
This is true. We can make it secure I think

Reply:
Is this something which should be handled in this PR? I agree that we should do it but I don't think it belongs here.

Reply:
Not necessarely, but we should not forget it. No need to sign the script if we keep this

Reply:
Feel free to open an issue for it.